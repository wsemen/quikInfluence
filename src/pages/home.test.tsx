import { render } from '@testing-library/react'
import Home from '.'

it('renders Home Page', () => {
  const { container } = render(<Home />)
  expect(container).toMatchSnapshot()
});
