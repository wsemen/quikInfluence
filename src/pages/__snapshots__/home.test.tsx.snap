// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Home Page 1`] = `
<div>
  <div
    class="container"
  >
    <main
      class="main"
    >
      <h1
        class="title"
      >
        Welcome to 
        <a
          href="#"
        >
          Quik Influence!
        </a>
      </h1>
      <p
        class="description"
      >
        Get started by logging in 
      </p>
      <div
        class="grid"
      >
        <a
          class="chakra-link card css-0"
          href="/dashboard"
        >
          <h2>
            Get Started â†’
          </h2>
        </a>
      </div>
    </main>
  </div>
</div>
`;
